File: Codechef/Taste_Dec/Taste_Dec.cpp
#include <iostream>
using namespace std;

int main()
{
    int T;
    cin >> T;
    while (T--)
    {
        int X, Y;
        cin >> X >> Y;
        if ((2 * X) > (5 * Y))
        {
            cout << "Chocolate" << endl;
        }
        else if ((2 * X) < (5 * Y))
        {
            cout << "Candy" << endl;
        }
        else
        {
            cout << "Either" << endl;
        }
    }
    return 0;
}

File: Codechef/Taste_Dec/Taste_Dec.py
T = int(input())
while T:
    T -= 1
    X, Y = map(int, input().split())
    if ((2 * X) > (5 * Y)):
        print("Chocolate")
    elif((2 * X) < (5 * Y)):
        print("Candy")
    else:
        print("Either")